directions.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+
+<html>
+
+	<head>
+		<title>About Us</title>
+	</head>
+
+	<body>
+
+	<h1>Welcome to the Bundle App Store</h1>
+	<p>
+	  This store brings you great app bundles week after week. We select the best power 
+      user apps from a broad range of suppliers and combine them into great deals. These are the highest quality apps form the best publishers, at great prices.
+	</p>
+
+	<p>
+		<a href="file:///C:/Users/Brendan%20Lyng/Documents/Red%20Hat/GitHub/Website/index.html">Link to Home Page</a>
```

Comment:
The link here isn't exactly right. It links to the local path of the file. You can use a relative path rather than the absolute path.